src/testdrive/src/action/sql.rs

```diff
@@ -162,7 +162,7 @@ pub async fn run_sql(mut cmd: SqlCommand, state: &mut State) -> Result<ControlFl
                         .collect::<Vec<_>>()
                         .join("\n");
 
-                    bail!("the on-disk state of the catalog does not match its in-memory state:\n{diff}");
+                    bail!("the in-memory state of the catalog does not match its on-disk state:\n{diff}");
```

Comment:
Unrelated change?

Reply:
It's related in the sense that this message confused me while I was debugging test failures related to this change. The order of words in the message now matches the order of strings being diffed.